portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -135,6 +150,11 @@
     <td>//div[@class='criteria-statement']//div[@class='input-group-item']/input</td>
     <td></td>
 </tr>
+<tr>
+    <td>SELECT_DOCUMENT</td>
+    <td>//div[@class='table-title text-truncate' and text()='${key_titleName}']</td>
```

Comment:
hey @ViniLopes20 , try to avoid composed classes (multiple classes with empty space between), this way your selector will be very fragile. If you are using contains, you don't need to add multiple classes ;)